com.unity.render-pipelines.universal/Runtime/Passes/PostProcessPass.cs

```diff
@@ -1391,6 +1399,122 @@ void RenderFinalPass(CommandBuffer cmd, ref RenderingData renderingData)
 
             var colorLoadAction = cameraData.isDefaultViewport ? RenderBufferLoadAction.DontCare : RenderBufferLoadAction.Load;
 
+            bool isFxaaEnabled = (cameraData.antialiasing == AntialiasingMode.FastApproximateAntialiasing);
+
+            if (cameraData.imageScaling != ImageScaling.None)
+            {
+                // FSR is only considered "enabled" when we're performing upscaling. (downscaling uses a linear filter unconditionally)
+                bool isFsrEnabled = ((cameraData.imageScaling == ImageScaling.Upscaling) && (cameraData.upscalingFilter == ImageUpscalingFilter.FSR));
+
+                // When FXAA is enabled in scaled renders, we execute it in a separate blit since it's not designed to be used in
+                // situations where the input and output resolutions do not match.
+                // When FSR is active we perform color conversion as part of the setup blit.
+                bool isSetupRequired = (isFxaaEnabled || isFsrEnabled);
+
+                // Make sure to remove any MSAA and attached depth buffers from the temporary render targets
+                var tempRtDesc = cameraData.cameraTargetDescriptor;
+                tempRtDesc.msaaSamples = 1;
+                tempRtDesc.depthBufferBits = 0;
+
+                m_Materials.upscaleSetup.shaderKeywords = null;
+
+                var sourceRtId = m_Source;
+
+                if (isSetupRequired)
+                {
+                    if (isFxaaEnabled)
+                    {
+                        m_Materials.upscaleSetup.EnableKeyword(ShaderKeywordStrings.Fxaa);
+                    }
+
+                    if (isFsrEnabled)
+                    {
+                        m_Materials.upscaleSetup.EnableKeyword(ShaderKeywordStrings.Gamma20);
+                    }
```

Comment:
See: https://github.com/Unity-Technologies/Graphics/pull/6178#discussion_r762097338

Also, I did add the fast path back into this commit. It may be worth keeping since it does seem to trigger on some projects like Viking Village by default.